### ISSUE TITLE
Update documentation for Express version 4

### ISSUE THREAD

**Issue by adonomay** [mcartoixa/connect-tedious#10]
The readme instructs developers to pass express to connect-tedious:
```javascript
var express = require('express');
var TediousStore = require('connect-tedious')(express);
```
In version 4.x, [Express removed all built-in middleware from its core except express.static](http://expressjs.com/guide/migrating-4.html#core-changes).  So the documentation should read:

```javascript
var express = require('express');
var session = require('express-session');
var TediousStore = require('connect-tedious')(session);
```

Not sure if you still want to support Express 3 -- at the very least include a note in the readme about Express 4 and express-session?

**Comment by mcartoixa** [comment]
Thanks for the feedback. We support both frameworks (at least for now) but I will update the documentation as you suggest.
I will close the issue when the next version is released (nothing planned yet).

**Comment by mcartoixa** [comment]
Landed in version 1.0.0
